<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionar Doctor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
      <item row="0" column="1">
       <widget class="QLineEdit" name="dc_nombre"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="dc_edad">
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDateEdit" name="dc_fecha_nacimiento"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Edad</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbtn_masculino">
           <property name="text">
            <string>Masculino</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="rbtn_femenino">
           <property name="text">
            <string>Femenino</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sexo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Fecha de  Nacimiento</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>#Registro Profesional</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="dc_registro_profesional"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Especialidad</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="dc_especialidad"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Residente</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="dc_residente">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tabla_doctor">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_insertar">
        <property name="text">
         <string>Insertar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_actualizar">
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_eliminar">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cerrar">
        <property name="text">
         <string>Cerrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dc_nombre</tabstop>
  <tabstop>rbtn_masculino</tabstop>
  <tabstop>rbtn_femenino</tabstop>
  <tabstop>tabla_doctor</tabstop>
  <tabstop>btn_insertar</tabstop>
  <tabstop>btn_actualizar</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_cerrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
